lib/cli.js

```diff
@@ -136,13 +137,49 @@ async function handleStdIn(argv) {
         column: parseInt(columnNumber)
       });
 
+      if (!firstFrame) {
+        firstFrame = res;
+      }
+
       // TODO: Fix formatting of the reoutputted stacktrace.
       result = result.replace(
         source,
         `  at ${functionName || ''} (${res.url}:${res.line}:${res.column})`
       );
     }
 
+    if (context) {
+      let { url } = firstFrame;
+
+      if (/^file:/.test(url)) {
+        url = pathModule.relative(process.cwd(), urlTools.fileUrlToFsPath(url));
+      } else if (/^https?:/.test(url)) {
+        let { pathname } = urlTools.parse(url);
+
+        if (pathname) {
+          url = pathModule.relative(process.cwd(), pathname.slice(1));
+        }
```

Comment:
This is the controversial bit.

For my sourcemaps I have no content, and the urls erroneously resolve to fake paths on a CDN. Attempting to resolve the pathname relative to the current working directory allows me to get the context anyway from my local checkout.

I could rewrite the urls to full `file://` urls and allow things like `CMD+Click` to open in macOS Terminal, for these cases as well.

Reply:
Hmm, yeah, it looks a little weird. Not sure I understand completely. It sounds like it's specific to some oddities with the source maps in your app?

Could you show an example of the original url and what you'd like to rewrite it to?

Any chance it could be generalized into a feature where you map a url prefix to a local "workspace" dir? Something like what I listed under https://github.com/papandreou/trackingdog#future-ideas

Reply:
I'd be happy if it could be accomplished via something like:

```
trackingdog --workspace https://mail-static.cdn-one.com/=.
```